CHANGES.md

```diff
@@ -0,0 +1,3 @@
+# sparsevec 0.1.0 (2018-10-16)
```

Comment:
Should be `2018-10-17`?

Reply:
I made the 0.1.0 release yesterday, but only pushing the crate today. So I don't think the date needs updating, but either works for me.

Reply:
I think the date should match the day we make the crate public (otherwise crates.io and CHANGES.md will show the thing being published on different days).

Reply:
Okay, that makes sense.